app/javascript/packs/diagnosis/content_form.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+    <div id="content-form">
+        <div class="ui dimmable segment shadow-less" v-bind:class="{ dimmed: disabled }">
+            <div class="ui simple inverted dimmer">
+                <div class="ui text loader">{{ pageData.locales.save_loader }}</div>
+            </div>
+            <div class="ui form">
+                <div class="field">
+                    <label>{{ pageData.locales.description }}</label>
+                    <textarea rows="2" v-model="diagnosis.content"></textarea>
+                </div>
+                <button class="small ui button" v-on:click="saveButtonClicked">
+                    {{ pageData.locales.save_description }}
+                </button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import axios from 'axios'
+
+    export default {
+        name: 'content-form',
+        data () {
+            return {
+                diagnosis: { content: ''},
+                disabled: true,
+                pageData: {
+                    diagnosis_id: '',
+                    locales: {
+                        description: '',
+                        save_description: '',
+                        save_loader: ''
+                    }
+                }
+            }
+        },
+        mounted: function () {
+            this.mountPageData();
+        },
+        watch: {
+            pageData: function (val) {
+                console.log('diagnosis_id: ' + this.pageData.diagnosis_id );
+                if(val.diagnosis_id) {
+                    this.getDiagnosis();
+                }
+            }
+        },
+        methods: {
```

Comment:
We should move these functions elsewhere to test them more easily.
For example:
```
getDiagnosis: getDiagnosis(this.pageData.diagnosis_id),
updateDiagnosis: updateDiagnosis
```

And right before the `</script>` tag:

```
function getDiagnosis(id) {
    var that;
    that = this;
    axios.get('/api/diagnoses/' id '.json')
        .then(function (response) {
            that.$nextTick(() => {
                that.diagnosis = response.data;
                that.disabled = false;
            })
        })
        .catch(function (error) {
            console.log('axios.get error'  error);
        });
}
```